### PR TITLE
138206 setting up the infrastructure required for testing projects

### PR DESCRIPTION
**Pull request by Shrutika-Ingale**
- setting up the infrastructure required for integration tests.

### CODE REVIEW

File: Kmd.Momentum.Mea.Api.Tests/HealthCheckTests.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Kmd.Momentum.Mea.Test.Common;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace Kmd.Momentum.Mea.Api.Tests
```
Comment (amitxagarwal) [inline Kmd.Momentum.Mea.Api.Tests/HealthCheckTests.cs]: rename it to Integration.tests

File: Kmd.Momentum.Mea.Test.Common/TokenHelper.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Kmd.Momentum.Mea.Test.Common
+{
+    public class TokenHelper : ITokenHelper
```
Comment (amitxagarwal) [inline Kmd.Momentum.Mea.Test.Common/TokenHelper.cs]: this should be a class in the test project and need not to be  a separate project

File: Kmd.Momentum.Mea.Test.Common/TokenHelper.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        public async Task<string> GetToken()
+        {
+            var clientCredentialsConfiguration = new ClientCredentialsConfiguration
+            {
+                GrantType = "client_credentials",
+                ClientId = "4a7a4c73-f203-435e-b5c2-3cbba12f0285",
+                ClientSceret = "PTY@yujcbe/kW7/9A0CvouR[qDArIpd0",
```
Comment (amitxagarwal) [inline Kmd.Momentum.Mea.Test.Common/TokenHelper.cs]: why these here and then repeated again later

File: Kmd.Momentum.Mea.Api.Tests/HealthCheckTests.cs (hunk 1 above)
Comment (Shrutika-Ingale) [reply Kmd.Momentum.Mea.Api.Tests/HealthCheckTests.cs]: Okay

File: Kmd.Momentum.Mea.Test.Common/TokenHelper.cs (hunk 2 above)
Comment (Shrutika-Ingale) [reply Kmd.Momentum.Mea.Test.Common/TokenHelper.cs]: Okay

File: Kmd.Momentum.Mea.Test.Common/TokenHelper.cs (hunk 3 above)
Comment (Shrutika-Ingale) [reply Kmd.Momentum.Mea.Test.Common/TokenHelper.cs]: Removed

**Pull request merged**